src/components/Header/index.ts

```diff
@@ -0,0 +1 @@
+export { default as Header } from './Header';
```

Comment:
```suggestion
export { default } from './Header';
export * from './Header';
```
저는 이렇게 사용하고있는데 무시하셔도 됩니다